apollo-backend/src/test/java/io/logz/apollo/EnvironmentTest.java

```diff
@@ -67,4 +67,97 @@ public void testGetAllEnvironments() throws ApolloClientException {
         }
         assertThat(found).isTrue();
     }
+
+    @Test
+    public void testGetAllActiveEnvironments() throws ApolloClientException {
+
+        ApolloTestClient apolloTestClient = Common.signupAndLogin();
+
+        // Add environments
+        Environment testEnvironment = createAndSubmitEnvironment(apolloTestClient, false);
+        Environment activeTestEnvironment = createAndSubmitEnvironment(apolloTestClient);
+
+        // Get all environments should return everything
+        List<Environment> allEnvironments = apolloTestClient.getAllEnvironments();
+
+        boolean foundNotActiveEnvironment = allEnvironments.stream()
+                .anyMatch(environment -> environment.getId().equals(testEnvironment.getId()));
+        boolean foundActiveEnvironment = allEnvironments.stream()
+                .anyMatch(environment -> environment.getId().equals(activeTestEnvironment.getId()));
+
+        assertThat(foundNotActiveEnvironment).isTrue();
+        assertThat(foundActiveEnvironment).isTrue();
+
+
+        List<Environment> allActiveEnvironments = apolloTestClient.getAllActiveEnvironments();
+
+        // Inactive environments should be excluded from search results
+        foundNotActiveEnvironment = allActiveEnvironments.stream()
+                .anyMatch(environment -> environment.getId().equals(testEnvironment.getId()));
+        assertThat(foundNotActiveEnvironment).isFalse();
+
+        // Active environment should be present in response
+        Optional<Environment> environmentFromApi = allActiveEnvironments.stream()
+                .filter(environment -> environment.getId().equals(activeTestEnvironment.getId()))
+                .findFirst();
+
+        assertThat(environmentFromApi).isPresent();
```

Comment:
Why here checking the presence of the environment and not like the checks before?